.travis.yml

```diff
@@ -122,6 +123,7 @@ jobs:
       language: scala
       jdk: oraclejdk8
       before_install:
+        - CGO_ENABLED=1
```

Comment:
Do you want `export CGO_ENABLED=1` here? I assume you want it in the env, not just in the shell?